### ISSUE TITLE
Other player skins missing when use itag and.

### ISSUE THREAD

**Issue by boomrider199231** [md-5/iTag#14]
so first when i put itag in with the newest version it puts TagAPI in aswell is that meant to be happening. Next thing is when its Itag is in all other player skins on the server go to a steve skin your player skin stays but no one else can see your own skin. Im using spigot 1.8.7 #40 and the most recent version of itag and protocol. So i dont know.

**Comment by boomrider199231** [comment]
yes tagapi is meant to be installed but i still have the skins not showing up

**Comment by Techcable** [comment]
Use iTag without TagAPI.

**Comment by binka-dev** [comment]
I have the same issue but I don't have TagAPI installed?

**Comment by HeroCC** [comment]
#13 will fix this
